tests/matrix_free/parallel_multigrid_adaptive_08.cc

```diff
@@ -342,7 +342,7 @@ do_test(const DoFHandler<dim> &dof)
        ++level)
     {
       smoother_data[level].smoothing_range     = 15.;
-      smoother_data[level].degree              = 5;
+      smoother_data[level].degree              = 6;
```

Comment:
this test change does not change the output?

Reply:
This is necessary after #7708 with the change in what the Chebyshev degree means. That PR was open while the test, added in #7696, was also open. So this simply fixes an only remotely related issue, see also here:
https://cdash.kyomu.43-1.org/viewTest.php?onlydelta&buildid=7762